docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,139 @@
+| December 28, 2022, 00:00:00  | December 28, 2022, 05:00:00  | December 28, 2022, 21:00:00  | December 28, 2022, 19:00:00  | December 29, 2022, 00:00:00  |
+| December 28, 2022, 00:00:00  | December 27, 2022, 16:00:00  | December 27, 2022, 08:00:00  | December 27, 2022, 05:00:00  | December 27, 2022, 10:00:00  |
+
+For example, you could create **Team Report Time (EST)** differently depending on the source:
+
+1. **Original Time**
+
+```
+convertTimezone([Original Time)], 'EST')
+```
+
+2. **Database Time** (say you know your database is in UTC)
+
+```
+convertTimezone([Database time (UTC)], 'EST', 'UTC')
+```
+
+3. **Metabase Report Time** (if you know that Metabase reports in PST)
+
+```
+convertTimezone([Metabase Report Time (PST)], 'EST', 'PST')
+```
+
+Once you've created **Team Report Time (EST)**, you could convert it again to a **Local Report Time (GMT)** (maybe for a teammate on the move):
+
+```
+convertTimezone([Team Report Time (EST)], 'GMT', 'EST')
+```
+
+Before you do time zone conversions, make sure you know the source time zone that you're working with. The example here shows you how the source time zones can be different depending on the system that it comes from (original event capture, database, Metabase), but your datasets might not make it so obvious. 
+
+When you make a custom column with a new time zone, it's usually a good idea to name the column with that time zone. We promise that this a life a lot easier when someone inevitably asks why certain numbers don't match.
+
+## Accepted data types
+
+| [Data type](https://www.metabase.com/learn/databases/data-types-overview#examples-of-data-types) | Works with `convertTimezone`  |
+| ----------------------- | -------------------- |
+| String                  | ❌                   |
+| Number                  | ❌                   |
+| Timestamp               | ✅                   |
+| Boolean                 | ❌                   |
+| JSON                    | ❌                   |
+
+This table uses `timestamp` and `datetime` interchangeably. If your dates and times are stored as strings or numbers in your database, you can [cast them to datetimes](../data-modeling/metadata-editing#casting-to-a-specific-data-type) from the Data Model page.
+
+## Related functions
+
+This section covers functions and formulas that work the same way as the Metabase `convertTimezone` expression, with notes on how to choose the best option for your use case.
+
+- [SQL](#sql)
+- [Spreadsheets](#spreadsheets)
+- [Python](#python)
+
+### SQL
+
+When you run a question using the [query builder](https://www.metabase.com/glossary/query_builder), Metabase will convert your graphical query settings (filters, summaries, etc.) into a query, and run that query against your database to get your results.
+
+If our [datetime sample data](#creating-custom-report-dates) is stored in a PostgreSQL database:
+
+```sql
+SELECT original_time::TIMESTAMP AT TIME ZONE 'EST' AS team_report_time_est
```

Comment:
it's not clear to me what is the type of `original_time` is it a timestamptz or timestamp type?